packages/electron-publish/src/gitHubPublisher.ts

```diff
@@ -215,6 +215,20 @@ export class GitHubPublisher extends HttpPublisher {
       })
   }
 
+  private doesErrorMeanAlreadyExists(e: any) {
+    let doesErrorMeanAlreadyExists = true
+    doesErrorMeanAlreadyExists = doesErrorMeanAlreadyExists && e.statusCode === 422
+    if (e.description) {
+      const desc = e.description
+      const descIncludesAlreadyExists =
+        (desc.includes("errors") && desc.includes("already_exists")) || (desc.errors && desc.errors.length >= 1 && desc.errors[0].code === "already_exists")
+      doesErrorMeanAlreadyExists = doesErrorMeanAlreadyExists && descIncludesAlreadyExists
```

Comment:
We can just simplify the logic flow here to be less nested (instead of refactoring after)
```
private doesErrorMeanAlreadyExists(e: any) {
  if (!e.description) {
  	return false
	}
  const desc = e.description
  const descIncludesAlreadyExists = (desc.includes("errors") && desc.includes("already_exists")) || (desc.errors && desc.errors.length > 0 && desc.errors[0].code === "already_exists")
	return e.statusCode === 422 && descIncludesAlreadyExists
}
```

Reply:
yeah, that might make more sense that way, a bit easier to follow at least

Reply:
I've fixed it up to be more like your suggestion